[GStreamer] REGRESSION(253289@main): Broke debug builds

philn opened the pull request:
#### ec1e02abd6f96d5aa790e6f969a2e3e3738c8ec5
<pre>
[GStreamer] REGRESSION(253289@main): Broke debug builds
<a href="https://bugs.webkit.org/show_bug.cgi?id=243781">https://bugs.webkit.org/show_bug.cgi?id=243781</a>

Reviewed by Adrian Perez de Castro.

The return type for the resizeTo() method was changed to RefPtr&lt;&gt;, which is more consistent with the
mac ImageTransferSessionVT::convertVideoFrame() signature.

* Source/WebCore/platform/graphics/gstreamer/VideoFrameGStreamer.cpp:
(WebCore::VideoFrameGStreamer::resizeTo):
* Source/WebCore/platform/graphics/gstreamer/VideoFrameGStreamer.h:

Canonical link: <a href="https://commits.webkit.org/253296@main">https://commits.webkit.org/253296@main</a>
</pre>

aperezdc reviewed (approved):
Nicer than #3190, thanks!

philn committed:
[GStreamer] REGRESSION(253289@main): Broke debug builds
https://bugs.webkit.org/show_bug.cgi?id=243781

Reviewed by Adrian Perez de Castro.

The return type for the resizeTo() method was changed to RefPtr<>, which is more consistent with the
mac ImageTransferSessionVT::convertVideoFrame() signature.

* Source/WebCore/platform/graphics/gstreamer/VideoFrameGStreamer.cpp:
(WebCore::VideoFrameGStreamer::resizeTo):
* Source/WebCore/platform/graphics/gstreamer/VideoFrameGStreamer.h:

Canonical link: https://commits.webkit.org/253296@main
